sw/device/silicon_creator/mask_rom/mask_rom_epmp_test.c

```diff
@@ -365,6 +366,10 @@ void mask_rom_main(void) {
       .sink = uart_sink,
   });
 
+  // Hack to ensure that sec_mmio_ctx is reachable and therefore kept
+  // in the .critical_static section.
+  sec_mmio_init(NULL);
```

Comment:
You can avoid this by writing `EXTERN(sec_mmio_ctx)` in the linker script. This forces the symbol to be looked up at link time.

Reply:
Thanks. I've ended up working around this a different way which is a bit cleaner I think.